unit_test.sh

```diff
@@ -13,7 +13,19 @@ mkdir -p test_results/latest
 DB_HOST_URL=${DB}?sslmode=disable
 TEST_DB_URL=${DB}/bcda_test?sslmode=disable
 echo "Running unit tests and placing results/coverage in test_results/${timestamp} on host..."
-DATABASE_URL=$TEST_DB_URL QUEUE_DATABASE_URL=$TEST_DB_URL gotestsum --junitfile test_results/${timestamp}/junit.xml -- -race ./... -coverprofile test_results/${timestamp}/testcoverage.out 2>&1 | tee test_results/${timestamp}/testresults.out
+
+# Avoid the db/migrations package since it only contains test code
+PACKAGES_TO_COVER=$(go list ./... | egrep -v 'test|mock|db/migrations' | tr "\n" "," | sed -e 's/,$//g')
+# Leverage the coverpkg flag to allow for coverage of packages even if there are tests outside of its package.
```

Comment:
Is this `coverprofile` instead of `coverpkg`?

Reply:
`coverprofile` writes the coverage numbers out to a file.

`coverpkg` allows tests in other packages to still count as "tested" for the listed packages. For example, our postgres package (before) did not have that many unit tests to verify the query functionality. However, we have E2E tests that use the actualy postgres repository. I was thinking it might be a good idea to apply coverage numbers of *any* of our tests called the function, but didn't want to do it w/o more team discussion.

Left in the unit_test.sh just so we have any easy hook if we decide to go for it.

Details here: https://golang.org/pkg/cmd/go/internal/test/

```
-coverpkg pattern1,pattern2,pattern3
        Apply coverage analysis in each test to packages matching the patterns.
        The default is for each test to analyze only the package being tested.
        See 'go help packages' for a description of package patterns.
        Sets -cover.
```